### PR TITLE
[cli] Remove unnecessary build outputs

### PR DESCRIPTION
**Pull request by TooTallNate**
Upon investigation into the `dist` dir, it appears that `ncc` is bundling some assets that don't need to be there. This change is a quick band-aid fix to remove those assets, without addressing the underlying cause of _why_ they're being bundled, which requires further investigation.

Overall about 1mb of disk space is saved.

### CODE REVIEW

**Review by lucleray — DISMISSED** [review]

**Review by styfle — COMMENTED** [review]
You might try `declaration: false` in `./packages/now-cli/tsconfig.json`

File: packages/now-cli/scripts/build.ts [hunk 1]
```diff
@@ -75,6 +75,17 @@ async function main() {
   const dest = join(dirRoot, 'dist/runtimes');
   await cpy('**/*', dest, { parents: true, cwd: runtimes });
 
+  // Band-aid to delete stuff that `ncc` bundles, but it shouldn't:
+
+  // TypeScript definition files from `@vercel/build-utils`
+  await remove(join(dirRoot, 'dist/dist'));
```
Comment (styfle) [inline packages/now-cli/scripts/build.ts]:
```suggestion
  await remove(join(dirRoot, 'dist', 'dist'));
```

File: packages/now-cli/scripts/build.ts [hunk 2]
```diff
@@ -75,6 +75,17 @@ async function main() {
...
+
+  // The Readme and `package.json` from "config-chain" module
+  await remove(join(dirRoot, 'dist/config-chain'));
```
Comment (styfle) [inline packages/now-cli/scripts/build.ts]:
```suggestion
  await remove(join(dirRoot, 'dist', 'config-chain'));
```

File: packages/now-cli/scripts/build.ts [hunk 3]
```diff
@@ -75,6 +75,17 @@ async function main() {
...
+
+  // A bunch of source `.ts` files from CLI's `util` directory
+  await remove(join(dirRoot, 'dist/util'));
```
Comment (styfle) [inline packages/now-cli/scripts/build.ts]:
```suggestion
  await remove(join(dirRoot, 'dist', 'util'));
```

**Comment by TooTallNate** [comment]
@styfle I applied your suggestions, but note that the change is a no-op (`path.join()` properly translates to forward slashes automatically).

**Comment by TooTallNate** [comment]
> You might try `declaration: false` in `./packages/now-cli/tsconfig.json`

@styfle Didn't help.

**Review by styfle — APPROVED** [review]

**Review by lucleray — APPROVED** [review]

**Pull request merged**